interface/forms/eye_mag/help.php

```diff
@@ -35,411 +34,572 @@
-                        The semi-colon "<strong>;</strong>" is used to divide entries, allowing multiple field entries simultaneously. <br />
-                        <small><i>The semi-colon separates entries.</i></small><br />
-                        After pressing <strong>Enter/Return</strong> or <strong>Tab</strong>, the data are submitted and the form is populated.  <br />
-                    </blockquote>
-                </div>
-            </div>
+                            <hr />
+                            <p>
+                                Below all these lines are strung together. Copy and paste this into a test patient's chart.
+                                <br />
+                                Voila! HPI, PMH, POH, Medications Allergies and 40 clinical findings + normals, are documented.
+                            </p>
+                            <hr />
 
-            <div id="accordion_hpi_group" name="accordion_group" class="ui-accordion" style="text-align:left;margin:10;padding:20;">
-                <div name="hpi" class="ui-accordion external">
-                    <h3 name="hpi_group" id="hpi_0">History of Present Illness: Shorthand Walk Through</h3>
-                    <div name="hpi_group" class="external" style="text-align:left;margin:0;padding:0;">
-                        <a name="example_hpi"></a>
-                        <blockquote class="style2">
-                            <h4 class="underline">Shorthand</h4>
-                            <textarea class="kb_entry"  style="min-height:1in;">CC:"My eyes are tearing and there is a yellow discharge";hpi:The symptoms began last week and the discharged turned yellow yesterday.  No photophobia.  The redness spread from the right to the left eye two days ago.;
+                            <textarea class="form-control" style="height:2.3in;">CC:"My eyes are tearing and there is a yellow discharge";hpi:The symptoms began last week and the discharged turned yellow yesterday.  No photophobia.  The redness spread from the right to the left eye two days ago.;
+                                POH:POAG. Myopia. Dry Eye; POS: Phaco/IOL OD 4/4/1994.Phaco/IOL OS 4/24/1995. Yag/PCO OD 6/5/1999;Meds:Timolol 0.5% GFS QHS OU. Latanoprost 0.01% QHS OU.
+                                Trazadone 50mg PO QHS.Famvir 500mg PO TID;Surg:Appendectomy 1998. Choly 2010.Lap Band 2014.;All:sulfa - hives.PCN - SOB;
+                                D;bll:+2 meibomitis;rll:frank ect, 7x6mm lid margin bcc lat.a;bul:2mm ptosis;rul:+3 dermato.a
+                                bc:+2 inj;bk:med pter;rk:mod endo gut.a;bac:+1 fc, +1 pig cells;
+                                bd:+2 bowtie pallor;rcup:0.6Vx0.4H w/ inf notch;lcup:0.5;rmac:+2 BDR, +CSME;lmac:flat, tr BDR;v:+PPDR, ++venous beading;rp:ht 1 o,no vh;
+                                scDist;5:8ix 1rht;4:10ix;6:6ix;2:15xt;8:5ix;ccDist;4:5ix;5:ortho;6:ortho
                             </textarea>
```

Comment:
@tywrenn  Should stay this style `style="height:2.3in;"` or remove it ?

Reply:
@stu01509 you should replace inches with rem, which in this example would be 13.8rem.

Reply:
I don't think we're gonna be able to replace the width/height inline styles too much cause some of them have to be specific and that would fill up an entire CSS file and make things harder to organize

Reply:
Okay, I adjusted to the rem unit.